### PR TITLE
[Qt] Pop debug window in foreground when opened twice

### PR DESCRIPTION
**Pull request by maflcko**
Currently the debug window stays in the background when opened a second time. It should pop up in the foreground, instead.

The commit is pretty much a conceptual copy of 5ffaaba and 382e9e2.

### CODE REVIEW

File: src/qt/bitcoingui.cpp [hunk 1]
```diff
@@ -352,6 +347,11 @@ void BitcoinGUI::createActions()
     connect(optionsAction, SIGNAL(triggered()), this, SLOT(optionsClicked()));
     connect(toggleHideAction, SIGNAL(triggered()), this, SLOT(toggleHidden()));
     connect(showHelpMessageAction, SIGNAL(triggered()), this, SLOT(showHelpMessageClicked()));
+    connect(openRPCConsoleAction, SIGNAL(triggered()), this, SLOT(showDebugWindow()));
+    // prevents an open debug window from becoming stuck/unusable on client shutdown
+    connect(quitAction, SIGNAL(triggered()), rpcConsole, SLOT(hide()));
+
+
```
Comment (Diapolo) [inline src/qt/bitcoingui.cpp]: Nit: Please remove one of the new-lines.

**Comment by Diapolo** [comment]
Seems to be another try like mine with #6288.

**Comment by jonasschnelli** [comment]
Tested ACK.
Binaries: https://builds.jonasschnelli.ch/pulls/6653/

**Comment by MarcoFalke** [comment]
@Diapolo Thanks for the feedback. I have rearranged the commits according to comments. The rename commit is now HEAD, so it can be easily dropped.

@jonasschnelli Thanks for the binaries. The binaries are using the (now outdated) ad3c945. I force pushed ed2c72e in the meantime. You can expect the following from ed2c72e:

 Debug window ...  | Mac OS | Fedora Linux (GNOME) | Windows 7
--------------------- | ---------  | ------------------------------ | -------------
... on another virtual desktop: | ? | switch to virtual desktop and show | n.a.
... hidden behind other windows: | ? | raise to foreground | raise to foreground 
... minimized: | ? | n.a. | show

**Comment by fanquake** [comment]
Tested  ed2c72e on OS X
```
On another virtual desktop: switch to virtual desktop and show
Hidden behind other windows: raise to foreground
Minimised: show
```

**Comment by MarcoFalke** [comment]
Trivial rebase (3-way merge)

**Comment by jonasschnelli** [comment]
ReACK.

**Pull request merged**